packages/ComponentLibrary/src/theme.ts

```diff
@@ -89,12 +89,12 @@ export const theme = createTheme({
       dark: SECONDARY_300,
       contrastText: PRIMARY_500,
     },
-    tertiary: {
-      light: TERTIARY_50,
-      main: TERTIARY_50,
-      dark: TERTIARY_100,
-      contrastText: PRIMARY_500,
-    },
+    // tertiary: {
```

Comment:
Use the colors defined in figma.

Reply:
I was trying to fix the build and the existing theme interface did not have "tertiary" prop and TS was complaining about it. I reverted back this change and updated the Theme and Palette type definitions

Reply:
This configuration was already working, that's why I asking.
Please check with S.Alaniz if this change touch other component.
@Sankku

Reply:
Yep I saw you've added the custom interface but somehow it got broken.
About this fix, we just checked together and he approved it

Reply:
Okay, well done !